apps/course-authoring/src/renderer/components/titlebar/titlebar.test.tsx

```diff
@@ -5,12 +5,12 @@ import { TitleBar } from './index';
 // Create routes that will be used to render the component
 const routes = [
   {
-    Route: '/',
-    Name: 'Dashboard',
+    PageRoute: '/',
+    PageName: 'Dashboard',
   },
   {
-    Route: '/create',
-    Name: 'Create New Course',
+    PageRoute: '/create',
```

Comment:
can you change this so that it imports the routes from the sub-pages

Reply:
hey @aubeOSG, I just came up with a solution to this bug. I pushed up my changes but this is not ready for review yet. Tomorrow morning I'll review my solution and I might make some changes.

I'll let you know when it's ready for review. Thanks!

Reply:
I fixed it directly in the app component by adding the `handleTitleChange` that will update the list of titles object and then re-render the title bar accordingly